handbook/technical-overview/devops/metrics.md

```diff
@@ -20,4 +16,70 @@ We are collecting metrics at three distinct levels:
 
 **Runtime** metrics are gathered from our application runtime - NodeJS. These are about the utilization of different types of heap, CPU, etc.
 
-**Application** metrics are about the entities and operations performed on those, that are in the domain of the application. Providing these metrics is a responsibility of the developers. It is important to have metrics for both successful as well as unsuccessful operations. For example: "application registration successful", application registration failed", etc. For an example, please take a look at this [file](https://github.com/island-is/island.is/blob/main/libs/infra-express-server/src/lib/infra-express-server.ts).
+**Application** metrics are about the entities and operations performed on those that are in the domain of the application. Providing these metrics is a responsibility of the developers. It is important to have metrics for both successful as well as unsuccessful operations. For example: "application registration successful", application registration failed", etc. For an example, please take a look at this [file](https://github.com/island-is/island.is/blob/main/libs/infra-express-server/src/lib/infra-express-server.ts).
+
+## Types of application metrics
+
+We are collecting application metrics of two types:
+
+- counters
+- timers
+
+**Counters** are used to count the number of times an event has occurred. For example: "number of successful HTTP requests", "number of failed HTTP requests", etc.
+
+**Timers** are used to measure the time it takes to perform an operation in milliseconds. For example: "time it takes to generate a response", "time it takes to generate a PDF", etc.
+
+### Naming metrics
+
+Metrics should be named with ASCII letters, underscores and periods. Periods are used to create a hierarchy that always starts with `islandis.`, provided by metrics client. Following that there should be a prefix for the given service and more detailed name for the metric in question. See more in ["What best practices are recommended for naming metrics and tags?" from DataDog](https://docs.datadoghq.com/developers/guide/what-best-practices-are-recommended-for-naming-metrics-and-tags/).
```

Comment:
A bit confusing that the example below does not follow the documented hierarchy, because the DogStatsD class automatically adds the "islandis" prefix. Worth explaining.